### ISSUE TITLE
GPML vs. wpTypes vocab

### ISSUE THREAD

**Issue by ariutta** [PathVisio/GPML#12]
ArrowHead type names are not consistent across our projects. For just one example:

~~wp vocab: [`Binding`](https://vocabularies.wikipathways.org/wp#Binding)~~
wpTypes vocab: [`MimBinding`](https://vocabularies.wikipathways.org/wpTypes#MimBinding)
Actual GPML: `mim-binding`

```xml
  <Interaction>
    <Graphics ZOrder="12288" LineThickness="1.0">
      <Point X="419.0" Y="208.0" GraphRef="e9078" RelX="-1.0" RelY="0.0" />
      <Point X="160.75" Y="103.25" GraphRef="f2588" RelX="0.0" RelY="1.0" ArrowHead="mim-binding" />
      <Anchor Position="0.4" Shape="None" />
    </Graphics>
    <Xref Database="" ID="" />
  </Interaction>
```

If the purpose of the wpTypes vocab is to help keep our projects harmonized, it needs to be in sync with actual GPML.

**Comment by mkutmon** [comment]
Yes! In the wp vocab it's different because it's the real semantics described there (so if we would support sbgn then sbgn-binding would also be mapped to WP vocab Binding).
As said - I didn't look at the WPType vocab in detail yet but there is definitely still lots of work to do! And yes it should be the same as in the GPML!

**Comment by ariutta** [comment]
Thanks @mkutmon! We can either change wpTypes to match PathVisio's GPML, or we can use this GPML update to make PathVisio's GPML reflect what's in wpTypes. Do you have a preference?

**Comment by ariutta** [comment]
Does this issue belong in the PathVisio/pathvisio repo? The 2013a xsd doesn't reference any of the mim types.
